test/pkg/environment/aws/environment.go

```diff
@@ -106,6 +111,12 @@ func NewEnvironment(t *testing.T) *Environment {
 		ClusterName:     lo.Must(os.LookupEnv("CLUSTER_NAME")),
 		ClusterEndpoint: lo.Must(os.LookupEnv("CLUSTER_ENDPOINT")),
 	}
+
```

Comment:
Do we have anonymous pull on the pull through cache if a private cluster is being used for testing locally?